scripts/scil_surface_from_labels.py

```diff
@@ -0,0 +1,141 @@
+                             assert_outputs_exist,
+                             ranged_type)
+
+EPILOG = """
+References:
+[1] St-Onge, E., Daducci, A., Girard, G. and Descoteaux, M. 2018.
+    Surface-enhanced tractography (SET). NeuroImage.
+"""
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(description=__doc__, epilog=EPILOG,
+                                formatter_class=argparse.RawTextHelpFormatter)
+
+    p.add_argument('in_label',
+                   help='Path of the volume (nii or nii.gz).')
+
+    p.add_argument('out_surface',
+                   help='Output surface (.vtk)')
+
+    p.add_argument('--indices', type=int, nargs='+',
+                   help='List of labels indices to use for the surface.')
+
+    p.add_argument('--value', default=0.5,
+                   type=ranged_type(float, 0, None, min_excluded=False),
+                   help='Isosurface threshold value used. '
+                        'This value is called isovalue '
+                        'in mbcube. [%(default)s]')
+
+    # Need a group here
+    morpho_g = p.add_argument_group('Morphology options')
+    morpho_g.add_argument('--smooth', default=0.0,
+                          type=ranged_type(float, 0, None, min_excluded=False),
+                          help='Smoothing size with'
+                               ' 1 implicit step. [%(default)s]')
+    morpho_g.add_argument('--erosion', default=0,
+                          type=ranged_type(int, 0, None, min_excluded=False),
+                          help='Erosion: number of iterations. [%(default)s]')
+    morpho_g.add_argument('--dilation', default=0,
+                          type=ranged_type(int, 0, None, min_excluded=False),
+                          help='Dilation: number of iterations. [%(default)s]')
+    morpho_g.add_argument('--opening', default=0,
+                          type=ranged_type(int, 0, None, min_excluded=False),
+                          help='Opening: number of iterations. [%(default)s]')
+    morpho_g.add_argument('--closing', default=0,
+                          type=ranged_type(int, 0, None, min_excluded=False),
+                          help='Closing: number of iterations. [%(default)s]')
+
+    p.add_argument('--fill', action='store_true',
+                   help='Fill holes in the image.')
+
+    p.add_argument('--vox2vtk', action='store_true',
+                   help='Transformation to vox2vtk.')
+
+    add_verbose_arg(p)
+    add_overwrite_arg(p)
+
+    return p
+
+
+def main():
+    parser = _build_arg_parser()
+    args = parser.parse_args()
+    logging.getLogger().setLevel(logging.getLevelName(args.verbose))
+
+    assert_inputs_exist(parser, args.in_label)
+    assert_outputs_exist(parser, args, args.out_surface)
+
+    # Load volume
+    labels_img = nib.load(args.in_label)
+    labels_volume = get_data_as_labels(labels_img)
+
+    # Removed indices
+    mask = get_binary_mask_from_labels(labels_volume, args.indices)
```

Comment:
As of now, this is crashing unless given (so I would use required at least). 
But since this is a script <from_labels> I think the default should be:
generate all the labels (from np.unique() then the script does the for loop itself) into an output folder, if the user wants just a few then --indices uses just one.
similar to the split labels script we could use an --out_dir OR --out_suffix .

This is my recommendation because if you run this on Freesurfer atlas, most people don't know the labels one by one and they probably want to generate a bunch of independent surface. And if they want to merge a few labels into a single mask, I propose that @arnaudbore you reuse the same logic as in scil_tractogram_filter_with_roi.py (like --indices 1 2 3 4 (do only 4 surfaces instead of all) VS --merge_indices 1:6 9 10:15 (do a single surface from a bunch of labels) and these two would be mutually exclusive options (default being all labels as multiple independent surfaces)

Reply:
Here is my trade off: if you want 4 surfaces you use a loop otherwise it's now working as in scil_tractogram_filter_with_roi.py and combine every single indices provided with the possibility of using 1:6 10 22

Reply:
How does it work if you want all surfaces from Desikan? You have to know and write 
`for i in range {7..72} {1001..1035} {2001..2035}; do scil_surface_from_labels.py ...; done` and then get around 50 errors (some values do not exist) ?

I think it needs a `--do_all_labels_indepependantly` kind of option (which implies providing no option for indices, or at least ignores it) and a for loop in Python.

Not all atlas are contiguous (so very hard to do with {1..10}), some atlas have hundreds of labels (long to list) and it's kind of annoying to have a labels_list.txt (like in connectoflow) on hand to do `for i in $(cat labels_list.txt); do scil_surface_from_labels.py ...; done`, especially if you have a custom created atlas or kind of random values.

Reply:
Done it should the way you wanted now.